docs/Plugins.rst

```diff
@@ -454,6 +492,13 @@ using Permit Fats again while the list is filtered.
 Game interface
 ==============
 
+.. _nopause:
+
+nopause
+-------
+Disables pausing (both manual and automatic) with the exception of pause forced
+by `reveal` ``hell``. This is nice for digging under rivers.
```

Comment:
No, it doesn't - you can't nest .rst formatting.  You can do the following, but it's relatively clumsy in raw text.  I went with the existing format because I wasn't sure people would use this one correctly:

    `reveal hell <reveal>`

...links the text "reveal hell" to the "reveal" link target.  Whitespace matters, so I tended to go with 'just put backticks around the command name' to make links.